pkg/cmd/server/kubernetes/node/node.go

```diff
@@ -252,40 +248,10 @@ func (c *NodeConfig) RunKubelet() {
 		}
 	}
 	if clusterDNS != nil && !clusterDNS.IsUnspecified() {
-		c.KubeletServer.ClusterDNS = []string{clusterDNS.String()}
+		return []string{clusterDNS.String()}
 	}
 
-	// only set when ContainerRuntime == "docker"
-	c.KubeletDeps.DockerClientConfig = c.DockerClientConfig
```

Comment:
this is not needed anymore?

Reply:
> this is not needed anymore?

It gets wired from options.  Some people were doing naughty things that prevented collapsing onto kube, but they all seem to be related to four integration tests.  Three of which are no longer required.

The process for creating similar testing hacks will now be, "get your patch into upstream kube, then you can use it here" instead of "hack it however you like".